.cirrus/Dockerfile

```diff
@@ -15,9 +15,14 @@ FROM ${CIRRUS_AWS_ACCOUNT}.dkr.ecr.eu-central-1.amazonaws.com/base:j17-latest
 
 USER root
 
-RUN apt-get update -y
-RUN curl -sL https://deb.nodesource.com/setup_18.x | bash -
-RUN apt-get install -y nodejs libicu-dev
+RUN apt-get update -y && \
```

Comment:
Thanks for following Docker [best practices](https://docs.docker.com/develop/develop-images/instructions/#apt-get)!!